p2p/src/authenticated/lookup/actors/listener.rs

```diff
@@ -127,15 +227,8 @@ impl<E: Spawner + Clock + ReasonablyRealtime + Network + Rng + CryptoRng + Metri
             .await
             .expect("failed to bind listener");
 
-        // Loop over incoming connections as fast as our rate limiter allows
-        loop {
-            // Ensure we don't attempt to perform too many handshakes at once
```

Comment:
TODO: Add comment about why we no longer wait (attempting to do so many connections per X)

TL;DR if we are behind the flow, we'll build up a backlog that will be stale by the time we get to it.